weaviate/collection/grpc.py

```diff
@@ -572,19 +603,6 @@ def __init__(
     def _query(self) -> _GRPC:
         return _GRPC(self.__connection, self.__name, self.__tenant, self.__consistency_level)
 
-    def _struct_value_to_py_value(self, value: _StructValue) -> _PyValue:
-        if isinstance(value, struct_pb2.Struct):
-            return {key: self._struct_value_to_py_value(value) for key, value in value.items()}
-        elif isinstance(value, struct_pb2.ListValue):
-            return [
-                self._struct_value_to_py_value(cast(_StructValue, value)) for value in value.values
-            ]
-        elif isinstance(value, str) or isinstance(value, float) or isinstance(value, bool):
-            return value
-        else:
-            assert value is None
-            return None
-
```

Comment:
Is this logic captured by the above changes now and/or is it redundant code?

Reply:
wasn't used anywhere, except by itself